kolibri/plugins/style_guide/assets/src/views/shell/table-of-content.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+
+  <ul>
+    <li v-for="anchor in anchors">
+      <a :href="route" @click="goToAnchor(anchor.hash)">
+        {{ anchor.label }}
+      </a>
+    </li>
+  </ul>
+
+</template>
+
+
+<script>
+
+  /**
+   * A component for auto-generating a table of content for a page. All elements
+   * with an [id] attribute defined will be shown in the TOC.
+   */
+
+  const map = require('lodash/map');
+
+  module.exports = {
+    name: 'TableOfContent',
```

Comment:
very minor, but maybe better to name the component as a plural: "tableOfContents"